charts/prometheus-elasticsearch-exporter/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: Elasticsearch stats exporter for Prometheus
 name: prometheus-elasticsearch-exporter
-version: 6.2.0
+version: 7.2.0
```

A minor change will do.
```suggestion
version: 6.3.0
```